### ISSUE TITLE
Crashing on Android when using firestack.analytics

### ISSUE THREAD

**Issue by ajonno** [fullstackreact/react-native-firestack#316]
iOS is fine. Android crashes immediately anytime this call is made:
```
firestack.analytics.logEventWithName("home_view")
      .catch(err => console.log('You should never end up here'));
```
am using:
```
"name": "react-native-firestack",
"version": "2.3.9",
```
react-native version:
```
"react-native": "0.41.1",
```

we're getting our app out to beta testers in 5 days(!) so appreciate any help to get this resolved.

crash log is:
![image](https://cloud.githubusercontent.com/assets/3005269/24125876/030ebf64-0e1f-11e7-9e58-1d488e4fe42e.png)

**Comment by joegoodall1** [comment]
We're getting the same error ^^^^^

Have you resolved this @ajonno?

**Comment by ajonno** [comment]
No, we've moved away from firestack and are using Google's Firebase JS API.

Get Outlook for iOS<https://aka.ms/o0ukef>
_____________________________

**Comment by Salakar** [comment]
@joegoodall1 @ajonno don't think this was even fully implemented on here, but there are much better [alternatives you can use](https://github.com/invertase/react-native-firebase).

**Comment by joegoodall1** [comment]
Thanks for replying @Salakar,

We've been looking at react-native-firebase as it looks a really good alternative but we've already done quite a lot of work to get Firestack working. How difficult/easy would it be to transition to react-native-firebase?

**Comment by ghuh** [comment]
I agree with @Salakar that the best solution is probably to switch to something supported like https://github.com/invertase/react-native-firebase. However, if you need a quick solution, just make sure to always send params with your analytics event and that should prevent this issue.
